packages/config/src/bin/dev.webpack.config.ts

```diff
@@ -40,6 +40,9 @@ const { plugins: externalPlugins = [], interceptChromeConfig, routes, ...externa
 
 const internalProxyRoutes: { [endpoint: string]: ProxyConfigArrayItem } = {
   ...routes,
+  '/apps/chrome': {
+    target: `http://127.0.0.1:${process.env.FEC_CHROME_PORT}`,
```

Comment:
It might be good to have this variable and use the `stage.foo.redhat.com` etc. hostnames instead as a default.

Reply:
Good point. I'll add it.